Disable fallocate #562992

This is only a workaround to fix FTBFS, the configure check is wrong and needs to be fixed:
http://www.amule.org/abugs/view.php?id=1572

--- a/src/ThreadTasks.cpp
+++ b/src/ThreadTasks.cpp
@@ -506,6 +506,9 @@ void CCompletionTask::OnExit()
 #include <stdlib.h>
 #include <errno.h>
 
+// #562992
+#undef HAVE_FALLOCATE
+
 CAllocateFileTask::CAllocateFileTask(CPartFile *file, bool pause)
 	// GetPrintable is used to improve the readability of the log.
 	: CThreadTask(wxT("Allocating"), file->GetFullName().RemoveExt().GetPrintable(), ETP_High),
